tests/conftest.py

```diff
@@ -412,7 +412,7 @@ def teardown():
         return None
 
 
-# @pytest.fixture(scope="session", autouse=True)
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
If you want to have it autouse enabled by default, let's either mark it by appropriate skip marker, or add statement like this in the beggining of the fixture:
```
 if config.ENV_DATA["platform"].lower() not in constants.MANAGED_SERVICE_PLATFORMS:
    return
```
To make it clear that it won't do anything on non manged service platforms.

Having a quick if statement which ends a function is preferable over large if blocks, since one can easily miss when these block ends ...